partition_snapshot_reader.hh

```diff
@@ -341,7 +359,7 @@ private:
             return std::exchange(_next_row, {});
         } else {
             _no_more_rows_in_current_range = true;
-            auto mf = _reader.next_range_tombstone(*_current_ck_range, _last_entry, _last_rts, position_in_partition_view::for_range_end(*_current_ck_range));
+            auto mf = _reader.next_range_tombstone(*_current_ck_range, _last_entry, _last_rts, for_range_end_in_half_reversed(*_current_ck_range));
```

Comment:
Does this mean that the previous patches were just broken?

Reply:
They weren't, in the sense that given my killswitch commit, in this line `Reversing` is never true. If I was to remove all commits that make incremental changes in favor of these which make changes that are at each step consistent with the rest of the code, I guess I would need to squash them all.